process_pr.py

```diff
@@ -684,6 +687,10 @@ def process_pr(repo_config, gh, repo, issue, dryRun, cmsbuild_user=None, force=F
         signing_categories.add("tests")
         push_test_issue = True
     except: pass
+    if (not issue.pull_request) and re.match(CREATE_REPO, issue.title):
```

Comment:
Thsi section is only run for issues so no need to check for `not issue.pull_request`. Replace `not issue.pull_request` with `cmssw_repo` ( property file should be createdonly for cmssw issues)